lib/themes/reps/scripts/reps.js

```diff
@@ -11,7 +11,9 @@ jQuery(function ($) {
       $(this).parents('.expanded').toggleClass('open');
     });
     $('.expanded a').click(function () {
-      window.location = $(this).attr('href');
+      if (!$('this').hasClass('ext')) {
```

Comment:
I've tested and for me it's not working. But I think this version is ok:
if ($('this').hasClass('ext')) {
  window.location = $(this).attr('href');
}

Reply:
@libetho My bad, I had to remove quotation marks around the **this** variable!